e2e_test/source_inline/tvf/postgres_query.slt

```diff
@@ -21,12 +29,12 @@ CREATE TABLE test (
 );"
 
 system ok
-PGHOST=db PGPORT=5432 PGUSER=postgres PGPASSWORD=postgres PGDATABASE=cdc_test psql -c "
+PGHOST=$PGHOST PGPORT=$PGPORT PGUSER=$PGUSER PGPASSWORD=postgres PGDATABASE=source_test psql -c "
 INSERT INTO test SELECT generate_series(1, 100), true, 1, 1, 1, 1.0, 1.0, 1.0, '2021-01-01', '00:00:00', '2021-01-01 00:00:00', '2021-01-01 00:00:00 pst', 'text', 'varchar', '1 day', '{}', '\x01';
 "
 
 query II
-select * from postgres_query('db', '5432', 'postgres', 'postgres', 'cdc_test', 'select * from test where id > 90;');
+select * from postgres_query('$PGHOST', '$PGPORT', '$PGUSER', 'postgres', 'source_test', 'select * from test where id > 90;');
```

Comment:
```suggestion
select * from postgres_query('$PGHOST', '$PGPORT', '$PGUSER', '$PGPASSWORD', 'source_test', 'select * from test where id > 90;');
```

Reply:
Not changed?

Reply:
If we pass in `PGPASSWORD`, this is just an empty string. So it will result in test failure, because in the RW batch executor side, we will tell the client the pw is an empty string.

Let me try add it to ci source test config and see if that can fix it.

Reply:
Changed. Seems like is due to my own local config.